.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+# For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn test
```

Comment:
```suggestion
yarn lint-staged
```
I have no idea why we want to execute tests while committing. Let's check the code style instead.